lib/utils/pulsar_functions.c

```diff
@@ -0,0 +1 @@
+#include "pulsar_functions.h"
```

Comment:
You don't need this file. It's just a header include.   If you did need this for something, then it should be included in the CMakeLists.txt file, otherwise this file won't be compiled.

Reply:
funny you you say this, i was also wondering about this file but for safety i copied what's done for the VDIF utils. to be clear, you're suggesting i remove `pulsar_functions.c`?

Reply:
Yes remove `pulsar_functions.c` I think `vdif_functions.c` used to have something else in it - and we've just never removed that file.

Reply:
nice, will do.